contributors/devel/sig-testing/writing-good-e2e-tests.md

```diff
@@ -25,8 +25,17 @@ time, and delayed merges.
 
 #### Debuggability ####
 
-If your test fails, it should provide as detailed as possible reasons
-for the failure in its output. "Timeout" is not a useful error
+If your test fails, it should provide as detailed as possible reasons for the
+failure in its failure message. The failure message is the string that gets
+passed (directly or indirectly) to `ginkgo.Fail(f)`. That text is what gets
+shown in the overview of failed tests for a Prow job and what gets aggregated
+by https://go.k8s.io/triage.
+
+A good failure message:
+- identifies the test failure
```

Comment:
should we say whether text should not needs to include names of dynamic properties like pod names

Reply:
this comment actually conflicts with the later suggestion:

> It is good practice to include details like the object that failed some assertion in the failure message because

it seems like aggregation handles it all right. You can disregard the comment

Reply:
Correct. Aggregation simplifies the failure message. I have added a paragraph with a link to the relevant code.